src/command/dev/watcher.rs

```diff
@@ -142,21 +152,21 @@ impl SubgraphSchemaWatcher {
         let mut last_message = None;
         match &self.schema_watcher_kind {
             SubgraphSchemaWatcherKind::Introspect(introspect_runner_kind) => {
-                let poll_interval_secs = 1;
                 let endpoint = introspect_runner_kind.endpoint();
+                let polling_interval = self.polling_interval;
                 eprintln!(
                     "{}polling {} every {} {}",
                     Emoji::Listen,
                     &endpoint,
-                    poll_interval_secs,
-                    match poll_interval_secs {
+                    polling_interval,
+                    match polling_interval {
                         1 => "second",
                         _ => "seconds",
                     }
                 );
                 loop {
                     last_message = self.update_subgraph(last_message.as_ref())?;
-                    std::thread::sleep(std::time::Duration::from_secs(1));
+                    std::thread::sleep(std::time::Duration::from_secs(polling_interval as u64));
```

Comment:
not needed if it's already a u64

```suggestion
                    std::thread::sleep(std::time::Duration::from_secs(polling_interval));
```